### PR TITLE
Fix SCM loader for Capistrano v3.7.x

### PR DESCRIPTION
**Pull request by JulienBreux**
Because: https://github.com/capistrano/capistrano/tree/v3.7.0/lib/capistrano/scm

### CODE REVIEW

File: lib/capistrano/multiconfig.rb [hunk 1]
```diff
@@ -47,9 +47,6 @@
       load(file) if File.exists?(file)
     end
 
-    # Load SCM tasks
-    load "capistrano/#{fetch(:scm)}.rb"
```
Comment (1st8) [inline lib/capistrano/multiconfig.rb]: Shouldn't it now be `configure_scm`?
At least thats what used in https://github.com/capistrano/capistrano/blob/master/lib/capistrano/setup.rb#L30

**Comment by 1st8** [comment]
My face when I stumbled upon this problem while upgrading an ancient capistrano 2.x setup and finding your fix a second later. Thanks!

File: lib/capistrano/multiconfig.rb (hunk 1 above)
Comment (1st8) [inline lib/capistrano/multiconfig.rb]: Yep, it works with `configure_scm`, so I think it should be used instead.

**Comment by JulienBreux** [comment]
Please @railsware 😃

**Comment by ayanko** [comment]
v3.1.0 released

**Comment by 1st8** [comment]
So no `configure_scm` required? Okay...

**Comment by ayanko** [comment]
> So no configure_scm required? Okay...

Is it an issue?
To be honest I don't have right now time to review capistrano changes.
Is this PR was wrong?

**Comment by JulienBreux** [comment]
Big thanks!

**Pull request merged**